app/assets/stylesheets/alchemy/navigation.scss

```diff
@@ -49,7 +49,7 @@
         position: absolute;
         top: 0;
         left: $collapsed-main-menu-width - 1;
-        width: $main-menu-width + 1;
+        min-width: $main-menu-width + 1;
         padding-left: 4*$default-padding;
```

Comment:
We should have a `max-width` as well then. We do not want very long texts to span over the whole screen.

Reply:
Ok max-width has been set to 350px if that a sensible default.